processor/src/routes/operation.route.ts

```diff
@@ -78,8 +78,8 @@ export const operationsRoute = async (fastify: FastifyInstance, opts: FastifyPlu
     '/payment-intents/:id',
     {
       preHandler: [
-        opts.oauth2AuthHook.authenticate(),
-        opts.authorizationHook.authorize('manage_project', 'manage_checkout_payment_intents'),
+        // opts.oauth2AuthHook.authenticate(),
```

Comment:
is this commented out intentionally?

Reply:
thanks for the good catch
https://github.com/commercetools/connect-payment-integration-template/pull/52/commits/12e0ceb43ad0e5aabbe2472df3a8ce4815ad3f15